pkg/cmd/cli/cmd/newapp.go

```diff
@@ -845,3 +849,14 @@ func (r *configSecretRetriever) CACert() (string, error) {
 	}
 	return "", nil
 }
+
+func checkGitInstalled(w io.Writer) {
+	gitBinary := "git"
+	if goruntime.GOOS == "windows" {
+		gitBinary = "git.exe"
+	}
+	_, err := exec.LookPath(gitBinary)
+	if err != nil {
+		fmt.Fprintf(w, "warning: Cannot find git. Ensure that it is installed and in your path.\n")
```

Comment:
does this align w/ the formatting we use for other warnings like when we create a binary build?

it would also be nice if we could (concisely) also explain the implications of not finding git.